calibrate/src/views/components/TopBar.js

```diff
@@ -30,16 +37,37 @@ const logoStyle = {
   marginRight: ".4em",
 };
 
+const docsLinkWrapper = {
+  display: "flex",
+  alignItems: "center",
+  color: "#ffffff",
+  marginRight: "5px",
+  padding: "5px",
+  cursor: "pointer",
+  textDecoration: "none",
+};
+
 const TopBar = () => {
   return (
     <AppBar style={styles}>
-      <div style={logoContainer}>
-        <img
-          alt="airqo.net"
-          style={logoStyle}
-          src="https://res.cloudinary.com/drgm88r3l/image/upload/v1602488051/airqo_org_logos/airqo_logo.png"
-        />
-        <span style={{ fontSize: "20px" }}>AirQalibrate</span>
+      <div style={appBarWrapper}>
+        <div style={logoContainer}>
+          <img
+            alt="airqo.net"
+            style={logoStyle}
+            src="https://res.cloudinary.com/drgm88r3l/image/upload/v1602488051/airqo_org_logos/airqo_logo.png"
+          />
+          <span style={{ fontSize: "20px" }}>AirQalibrate</span>
+        </div>
+        <a
+          href="https://docs.google.com/document/d/1z65wAjDscZvA9i_oLyHaNm7xKjkaq_DFBd3vjapVzXA/edit?usp=sharing"
```

Comment:
Thanks @Codebmk, 
Is it possible for us to look into the possibilities of hosting this documentation on the newly set up documentation by  @123MwanjeMike ?

Reply:
Yeah. Informed Mike about this request.